plugins/experimental/uri_signing/unit_tests/CMakeLists.txt

```diff
@@ -26,8 +26,6 @@ add_executable(
   ${PROJECT_SOURCE_DIR}/match.cc
   ${PROJECT_SOURCE_DIR}/normalize.cc
   ${PROJECT_SOURCE_DIR}/parse.cc
-  ${PROJECT_SOURCE_DIR}/timing.cc
-  ${PROJECT_SOURCE_DIR}/uri_signing.cc
 )
 target_compile_definitions(uri_signing_test PRIVATE UNITTEST)
```

Comment:
Looks like the unit test stuff still needs some cleanup.  The -D for the unit test is URI_SIGNING_UNIT_TEST ( https://github.com/apache/trafficserver/blob/6e00344df420db067e2b3cddbe8a7a48b7559700/plugins/experimental/uri_signing/common.cc#L21 ), not UNITTEST.  But, since the test works, it looks like the code triggered by URI_SIGNING_UNIT_TEST can just be deleted.  But we could do all that in a separate PR.